### ISSUE TITLE
Make a real call to api/userProfile/<id> to save user Google email

### ISSUE THREAD

**Issue by alex-anakin** [hackforla/VRMS#473]
This task is blocked by:
- [x] #472
- [ ] implementation of `api/userProfiles` API

### Overview
Page was implemented with a call to dummy API. Replace it with the real call.

**Comment by jbubar** [comment]
no longer relevant
